addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/stripe.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yeelight" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="stripe">
+		<label>Yeelight Color LED Stripe</label>
+		<description>Yeelight Color LED Stripe</description>
+
+		<channels>
+			<channel id="brightness" typeId="brightness" />
```

Comment:
Please remove the brightness channel here as "color" already covers this functionality.